pkg/runutil/runutil.go

```diff
@@ -108,6 +115,12 @@ func CloseWithLogOnErr(logger log.Logger, closer io.Closer, format string, a ...
 	level.Warn(logger).Log("msg", "detected close error", "err", errors.Wrap(err, fmt.Sprintf(format, a...)))
 }
 
+// ExhaustCloseWithLogOnErr closes the io.ReadCloser with a log message on error but exhausts the reader before.
+func ExhaustCloseWithLogOnErr(logger log.Logger, readcloser io.ReadCloser, format string, a ...interface{}) {
+	var _, _ = io.Copy(ioutil.Discard, readcloser) // best effort
```

Comment:
No needed comment IMO. also kill `var`
```suggestion
	 _, _ = io.Copy(ioutil.Discard, readCloser) 
```

Reply:
Killed these :)